static/css/around-map.css

```diff
@@ -1,4 +1,4 @@
-.marker.you {
+.marker.current-location {
     background-image: url('/static/images/marker-red.png');
     z-index: 9999;
```

Comment:
Do not make a z-index that is so huge. I understand it should be over everything, but what if everything else like an alert banner needs to be over? And then a popup window over that banner alert?
Whe end up with ridiculous counts of _9_s.
Instead why don't you just apply a z-index on the markers (eg: `z-index: 2`), and 1 higher on the `.current-location`?